libexec/klab-build-spec

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env node
+
+const fs            = require("fs");
+const path          = require("path");
+const marked        = require("marked");
+const _             = require("lodash");
+const makeConfig    = require("../lib/config.js");
+const {
+  proofCollection,
+  parseAct,
+  buildActs,
+  caseSplitAct,
+  newCleanup,
+  makeInterabiExhaustiveness
+}                   = require("../lib/build.js");
+const {
+  ensureDirs,
+  read,
+  revert,
+  sha3,
+  testPath,
+  warn,
+}                   = require("../lib/util.js");
+const __a2n = act => act.subject + "_" + act.name;
+
+const KLAB_OUT = process.env.KLAB_OUT || "out";
+
+const usage = `
+Usage:
+  klab build-spec [<spec-name>]
+`
+
+if (!testPath("./config.json")) {revert("No config.json file in directory. Review `klab/examples` folder for inspiration")}
+const config_json   = JSON.parse(fs.readFileSync("./config.json"));
+const config        = makeConfig(config_json);
+const config_path   = config.src.specification;
+const spec_name     = process.argv[2];
+const raw_md_config = Array.isArray(config_path) && config_path.map(p => read(p)).join("\n\n") || read(config_path)
+
+config.get_proof_hash = ({name, spec}) => {
+  let proof = { spec : spec }
+  return sha3(JSON.stringify(proof))
+}
+
+// 1. get acts
+//    return a [behaviour]
+// 2. parse acts
+//    parses every behaviour into a structured object
+// 3. build acts
+
+const tokens        = marked.lexer(raw_md_config)
+const parsed_tokens = tokens
+  .map(t => {
+    if(t.type == 'code' && t.lang === "act") {
```

Comment:
Is there some reason to use `==` for one comparison but `===` for the other? Otherwise I'd say just use `===` for both.

Reply:
Hmmmm, these were just copy-pasted from `klab-build-js`, so not sure why they are how they are. I'll try it with `===` everywhere and see what happens.